facefusion/globals.py

```diff
@@ -54,3 +54,5 @@
 frame_processors : List[str] = []
 # uis
 ui_layouts : List[str] = []
+# ip camera url
+ip_camera_url : Optional[str] = None
```

Comment:
the webcam UI is and should not become part of the globals for now - remove this please